File: references/RndIndoorLaboratoryMasterDeleted.cs
using System;

namespace RnDLaboratoryStock.Core.Models;

public partial class RndIndoorLaboratoryMasterDeleted
{
    public int Id { get; set; }

    public int OldId { get; set; }

    public string? Epc { get; set; }

    public string ProductNumber { get; set; } = null!;

    public DateTime? ExpiredDate { get; set; }

    public string CabinetNumber { get; set; } = null!;

    public int Shelf { get; set; }

    public string Quantity { get; set; } = null!;

    public string Brand { get; set; } = null!;

    public string Code { get; set; } = null!;

    public string PackageQuantity { get; set; } = null!;

    public string? Unit { get; set; }

    public string? ChemicalName { get; set; }

    public string? SupplierName { get; set; }

    public string? QuantityNumber { get; set; }

    public int? Limit { get; set; }

    public int? ExpireLimit { get; set; }

    public DateTime InsertedAt { get; set; }

    public DateTime DeletedAt { get; set; }

    public string WmCode { get; set; } = null!;
}

File: app/src/main/java/com/example/rndlaboratorystock/Models/RndIndoorLaboratoryMasterDeleted.java
package com.example.rndlaboratorystock.Models;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class RndIndoorLaboratoryMasterDeleted {
    @SerializedName("responseCode")
    public int ResponseCode;
    @SerializedName("data")
    public List<Data> Data = null;

    public class Data {
        @SerializedName("id")
        public int Id;
        @SerializedName("oldId")
        public int OldId;
        @SerializedName("epc")
        public String Epc;
        @SerializedName("productNumber")
        public String ProductNumber;
        @SerializedName("expiredDate")
        public String ExpiredDate;
        @SerializedName("cabinetNumber")
        public String CabinetNumber;
        @SerializedName("shelf")
        public int Shelf;
        @SerializedName("quantity")
        public String Quantity;
        @SerializedName("brand")
        public String Brand;
        @SerializedName("code")
        public String Code;
        @SerializedName("packageQuantity")
        public String PackageQuantity;
        @SerializedName("unit")
        public String Unit;
        @SerializedName("chemicalName")
        public String ChemicalName;
        @SerializedName("supplierName")
        public String SupplierName;
        @SerializedName("quantityNumber")
        public String QuantityNumber;
        @SerializedName("limit")
        public int Limit;
        @SerializedName("expireLimit")
        public int ExpireLimit;
        @SerializedName("insertedAt")
        public String InsertedAt;
        @SerializedName("deletedAt")
        public String DeletedAt;
        @SerializedName("wmCode")
        public String WmCode;
    }
}
